README.md

```diff
@@ -56,7 +68,7 @@ Features of the native mobile Player SDKs are progressively being implemented in
 | -------------------------------- | ----------------------------------------- |
 | Playback of DRM-protected assets | :white_check_mark: Available since v0.2.0 |
 | Subtitles & Captions             | :white_check_mark: Available since v0.2.0 |
-| Advertising                      | :white_check_mark: Available since v0.4.0                |
+| Advertising                      | :white_check_mark: Available since v0.4.0 |
 | Playlist API                     | :x: Not available                         |
 | Offline Playback                 | :x: Not available                         |
 | Analytics                        | :x: Coming Q1 2023                        |
```

Comment:
@rolandkakonyi shouldn't Analytics be updated as well now 🙂

Reply:
@dweinber yes, we missed that on this PR but then updated here: https://github.com/bitmovin/bitmovin-player-react-native/commit/e78fe14c8a771c9d6ef0eae28c6b83f1780ce8f4

Reply:
Sorry, didn't check the current README, just stumbled upon this in the PR 🤦 thanks!